com.ashwingohil.petalssticky.json

```
@@ -0,0 +1,39 @@
+{
+    "app-id" : "com.ashwingohil.petalssticky",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
```

```suggestion
    "runtime-version" : "45",
```
It is the latest runtime